### DISCUSSION TITLE
Maximize pane

### DISCUSSION THREAD

**Discussion by TimoKramer** [wezterm/wezterm#2776] [Ideas]
Hi,
I've been using kitty for a long time now but switched to wezterm now because I need windows-support. Wezterm is awesome, thanks!

However there is one feat that I sorely miss, having a [pane layout](https://sw.kovidgoyal.net/kitty/conf/#shortcut-kitty.Next-layout). I mostly need it because on a small screen I want to switch between having a repl and a terminal beneath my nvim to fullscreen nvim and back. Is something like maximizing a certain pane possible with wezterm?

**Comment by wez** [comment]
Take a look at [TogglePaneZoomState](https://wezfurlong.org/wezterm/config/lua/keyassignment/TogglePaneZoomState.html), it is bound to CTRL-SHIFT-Z by default.

**Reply by TimoKramer** [reply]
Thank you!
